docs/experimenter/README.md

```diff
@@ -518,12 +519,13 @@ A live rollout can have updates pushed to its state while remaining Live. These
         Note over Experimenter Worker: Worker updates rollout
         Experimenter Worker->>Remote Settings Backend: Check collection (timeout)
         Experimenter Worker->>Experimenter Backend:  Status: Live <br/> Publish status: Idle <br/> Status next: <none> <br/> + changelog
+        Experimenter Backend->>Experimenter Backend: is_dirty: False 
```

Comment:
No, there's no need to explicitly set the value here.  It gets set when the write happens above and then stays there until it is successfully resolved back to False after the update in RS completes.

Reply:
We are showing `is_paused` on these diagrams, which is set in the same places as `is_dirty`. Should we remove `is_paused` from the diagrams as well? (I would prefer to keep both of them)

Reply:
oh wait I see what you mean. Misread this ;)